scripts/train.py

```diff
@@ -0,0 +1,181 @@
+from torch.optim import Adam
```

Comment:
The same concern about the order of imports here.

Reply:
Done